digraph "CFG forkotlinx.coroutines.ExceptionsKt.CancellationException(Ljava\/lang\/String;Ljava\/lang\/Throwable;)Ljava\/util\/concurrent\/CancellationException;" {
Node_4 [shape=record,label="{4\:\ ?|MTH_ENTER_BLOCK\l}"];
Node_6 [shape=record,label="{6\:\ 0x0000|java.util.concurrent.CancellationException r0 = new java.util.concurrent.CancellationException(r3)\lr0.initCause(r4)\l}"];
Node_7 [shape=record,label="{7\:\ 0x000a|RETURN\l|return r0\l}"];
Node_5 [shape=record,label="{5\:\ ?|MTH_EXIT_BLOCK\l}"];
MethodNode[shape=record,label="{public static final java.util.concurrent.CancellationException kotlinx.coroutines.ExceptionsKt.CancellationException((r3v0 'message' java.lang.String A[D('message' java.lang.String), IMMUTABLE_TYPE, METHOD_ARGUMENT]), (r4v0 'cause' java.lang.Throwable A[D('cause' java.lang.Throwable), IMMUTABLE_TYPE, METHOD_ARGUMENT]))  | USE_LINES_HINTS\lINLINE_NOT_NEEDED\lTYPE_VARS: EMPTY\l}"];
MethodNode -> Node_4;Node_4 -> Node_6;
Node_6 -> Node_7;
Node_7 -> Node_5;
}

